estuary/utils/story.py

```diff
@@ -195,9 +205,133 @@ def get_wait_times(self, results):
 
             # Find the time between when the current artifact completes and the next one starts
             wait_time = next_artifact_start_time - completion_time
-            wait_times[index] = wait_time.seconds
+            wait_times[index] = wait_time.total_seconds()
+
+            # The 'wait time' between a FreshmakerEvent and a ContainerKojiBuild is still a part of
+            # the processing in a FreshmakerEvent, so we do not count it towards the total wait time
+            if artifact.__label__ != 'FreshmakerEvent':
+                total_wait_time += wait_time.total_seconds()
+
+        return wait_times, total_wait_time
+
+    def get_total_processing_time(self, results):
+        """
+        Get the total time spent processing in the story.
+
+        :param list results: contains inflated results from Neo4j
+        :return: the seconds of total time spent processing
+        :rtype: int
+        """
+        total = 0
+        build = None
+        timed_processes = {
+            'FreshmakerEvent': ['time_created', 'time_done'],
+            'KojiBuild': ['creation_time', 'completion_time'],
+            'ModuleKojiBuild': ['creation_time', 'completion_time'],
+            'ContainerKojiBuild': ['creation_time', 'completion_time'],
+            'Advisory': ['created_at', 'status_time'],
+            'ContainerAdvisory': ['created_at', 'status_time']
+        }
+        for index, artifact in enumerate(results):
+            if artifact.__label__ not in timed_processes:
+                continue
+
+            creation_time = getattr(artifact, timed_processes[artifact.__label__][0])
+            if not creation_time:
+                id_num = getattr(artifact, artifact.unique_id_property + '_')
+                log.warning(
+                    'While calculating the total processing time, a %s with id %s was encountered '
+                    'without a creation time.',
+                    artifact.__label__, id_num
+                )
+                continue
```

Comment:
I'm wondering if the method should return `None` in this case since the total processing time will be inaccurate. Let's leave it for now, but could you reach out to Ralph about this?

Reply:
@sarah256 I forget where we landed on this. Could you please remind me?

Reply:
Oh yeah I talked to Ralph. He said "I'm okay with the current behavior. of skipping the stubbed artifact and having the total time be slightly off"